Neo4j.Driver/Neo4j.Driver/Internal/IO/PackStreamReader.cs

```diff
@@ -19,14 +19,40 @@
 using System.Collections.Generic;
 using System.IO;
 using Neo4j.Driver.Internal.Connector;
+using Neo4j.Driver.Internal.Messaging;
 
 namespace Neo4j.Driver.Internal.IO;
 
-internal sealed class PackStreamReader
+internal interface IPackStreamReader
```

Comment:
Need to undo, there is no use for the interface as the classes for Span and Sequence Readers are ref structs